librz/analysis/p/analysis_ppc_cs.c

```diff
@@ -209,208 +210,228 @@ static void opex(RzStrBuf *buf, csh handle, cs_insn *insn) {
+		"fpu	f10 .64 3328	0\n"
+		"fpu	f11 .64 3456	0\n"
+		"fpu	f12 .64 3584	0\n"
+		"fpu	f13 .64 3712	0\n"
+		"fpu	f14 .64 3840	0\n"
+		"fpu	f15 .64 3968	0\n"
+		"fpu	f16 .64 4096	0\n"
+		"fpu	f17 .64 4224	0\n"
+		"fpu	f18 .64 4352	0\n"
+		"fpu	f19 .64 4480	0\n"
+		"fpu	f20 .64 4608	0\n"
+		"fpu	f21 .64 4736	0\n"
+		"fpu	f22 .64 4864	0\n"
+		"fpu	f23 .64 4992	0\n"
+		"fpu	f24 .64 5120	0\n"
+		"fpu	f25 .64 5248	0\n"
+		"fpu	f26 .64 5376	0\n"
+		"fpu	f27 .64 5504	0\n"
+		"fpu	f28 .64 5632	0\n"
+		"fpu	f29 .64 5760	0\n"
+		"fpu	f30 .64 5888	0\n"
+		"fpu	f31 .64 6016	0\n"
+
+		// Vector registers (subset of vector scalar registers)
+		"vc	v0	.128 6144	0\n"
+		"vc	v1	.128 6272	0\n"
+		"vc	v2	.128 6400	0\n"
+		"vc	v3	.128 6528	0\n"
+		"vc	v4	.128 6656	0\n"
+		"vc	v5	.128 6784	0\n"
+		"vc	v6	.128 6912	0\n"
+		"vc	v7	.128 7040	0\n"
+		"vc	v8	.128 7168	0\n"
+		"vc	v9	.128 7296	0\n"
+		"vc	v10	.128 7424	0\n"
+		"vc	v11	.128 7552	0\n"
+		"vc	v12	.128 7680	0\n"
+		"vc	v13	.128 7808	0\n"
+		"vc	v14	.128 7936	0\n"
+		"vc	v15	.128 8064	0\n"
+		"vc	v16	.128 8192	0\n"
+		"vc	v17	.128 8320	0\n"
+		"vc	v18	.128 8448	0\n"
+		"vc	v19	.128 8576	0\n"
+		"vc	v20	.128 8704	0\n"
+		"vc	v21	.128 8832	0\n"
+		"vc	v22	.128 8960	0\n"
+		"vc	v23	.128 9088	0\n"
+		"vc	v24	.128 9216	0\n"
+		"vc	v25	.128 9344	0\n"
+		"vc	v26	.128 9472	0\n"
+		"vc	v27	.128 9600	0\n"
+		"vc	v28	.128 9728	0\n"
+		"vc	v29	.128 9856	0\n"
+		"vc	v30	.128 9984	0\n"
+		"vc	v31	.128 10112 0\n"
+
+		// Control registers
+		"ctr	xer .64 10240	0	# Fixed-Point Exception Register\n"
+		// 0 |---/32/---|--vrsave--| 63
+		"vcc	vrsave .32	10336	0	# VR Save Register\n"
+		"ctr	fpscr .64	10368	0	# Floating-Point Status and Control Register\n"
+		// 0 |-----------/96/-----------|--vscr--| 127
+		"vcc	vscr	.32	10528	0	# Vector Status and Control Register\n"
+		// 0 |---/32/---|----cr----| 63
+		"ctr	cr	.32	10592	0	# Condition Register\n"
+		"ctr	cr0 .4	10592	0	# Condition Register Field 0\n"
+		"ctr	cr1 .4	10596	0	# Condition Register Field 1\n"
+		"ctr	cr2 .4	10600	0	# Condition Register Field 2\n"
+		"ctr	cr3 .4	10604	0	# Condition Register Field 3\n"
+		"ctr	cr4 .4	10608	0	# Condition Register Field 4\n"
+		"ctr	cr5 .4	10612	0	# Condition Register Field 5\n"
+		"ctr	cr6 .4	10616	0	# Condition Register Field 6\n"
+		"ctr	cr7 .4	10620	0	# Condition Register Field 7\n"
+		"ctr	lr	.64	10624	0	# Link Register\n"
+		"ctr	ctr .64	10688	0	# Count Register\n"
+		"ctr	tar .64	10752	0	# Target Address Register\n"
+		"ctr	ppr32 .32	10816	0	# Process Priority Register 32-bit\n"
+		"ctr	ppr .64	10816	0	# Process Priority Register\n"
+		"ctr	dscr	.64	10880	0	# Data Stream Control Register\n"
+		"ctr	bescr .64	10944	0	# Branch Event Status and Control Register\n"
+		"ctr	ebbhr .64	11008	0	# Event-Based Branch Handler Register\n"
+		"ctr	ebbrr .64	11072	0	# Event-Based Branch Return Register\n"
+		"ctr	lpcr	.64	11136	0	# Logical Partitioning Control Register\n"
+		"ctr	hrmor .64	11200	0	# Hypervisor Real Mode Offset Register\n"
```

Comment:
many GPR registers are missing, like the dbat, ibat and hid registers.
by doing these changes you will break completely the implemented esil.

Reply:
I am aware of this. I will complete it in a few days when the first instructions get uplifted. Currently this PR is mainly for tracking the overall progress and not meant to be merged yet.